pandora-server-directory/src/spaces/space.ts

```diff
@@ -656,6 +700,7 @@ export class Space {
 
 		await this._assignedShard?.update('characters');
 		ConnectionManagerClient.onSpaceListChange();
+		await GetDatabase().updateSpace(this.id, { invites: this._invites }, null);
```

Comment:
Wouldn't it be better if `_cleanupInvites` did this asynchronously if the cleanup actually changed something? It can also be done asynchronously as it doesn't matter if the cleanup fails - it can just be repeated the next time. Only potential problem with that are race conditions...

Reply:
actually it could not be used in all the other cases when the clean up changed, since we either required to change or we save later anyway